code/game/turfs/simulated/walls.dm

```diff
@@ -240,7 +240,10 @@
 	proj_damage = round(Proj.get_structure_damage() / 3)//Yo may replace 3 to 5-6 to make walls fucking stronk as a Poland
 
 	//cap the amount of damage, so that things like emitters can't destroy walls in one hit.
-	var/damage = min(proj_damage, 100)
+	if(Proj.nocap_structures)
+		damage = proj_damage * 3
```

Comment:
The `var/damage` got deleted from above, so i wonder what `damage` this is referring to?

Reply:
It mistakenly referred to the amount of damage of the wall, the wall would first take extra damage and then reset to a semi-damaged state after every shot.